cerberus/client.py

```diff
@@ -55,6 +56,104 @@ def get_token(self):
         """Returns a client token from Cerberus"""
         return self.token
 
+    def get_roles(self):
+        """Returns all the roles (IAM or User Groups) that can be granted to a safe deposit box.
+
+        Roles are permission levels that are granted to IAM or User Groups.  Associating the id for the write role
+          would allow that IAM or User Group to write in the safety deposit box."""
+        sdb_resp = requests.get(self.cerberus_url + '/v1/role',
+                                headers=self.HEADERS)
+
+        self.throw_if_bad_response(sdb_resp)
+        return sdb_resp.json()
+
+    def get_role(self, key):
+        """Returns id of named role."""
+
+        sdb_resp = requests.get(self.cerberus_url + '/v1/role',
+                                headers=self.HEADERS)
+        self.throw_if_bad_response(sdb_resp)
+        json_resp = sdb_resp.json()
+        for item in json_resp:
+            if key in item["name"]:
+                return item["id"]
+        raise CerberusClientException("Key '%s' not found" % key)
+
+    def list_roles(self):
+        """Simplified version of get_roles that returns a dict of just name: id for the roles"""
+
+        json_resp = self.get_roles()
+        temp_dict = {}
+        for item in json_resp:
+            temp_dict[item["name"]] = item["id"]
+        return temp_dict
+
+    def get_categories(self):
+        """ Return a list of categories that a safe deposit box can belong to"""
+        sdb_resp = requests.get(self.cerberus_url + '/v1/category',
+                                headers=self.HEADERS)
+
+        self.throw_if_bad_response(sdb_resp)
+        return sdb_resp.json()
+
+    def create_sdb(self, name, category_id, owner, description="", user_group_permissions=None,
+                   iam_principal_permissions=None):
+        """Create a safety deposit box.
+
+        You need to get a new token before the iam role is granted permission to the new safety deposit box.
```

Comment:
I think we call it a "refresh" for the token.  Is that right?

Reply:
Yeah.  I can change the wording on that.